modules/configuring-private-storage-endpoint-azure-user-provided-vnet-subnet.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/configuring-private-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-private-storage-endpoint-azure-user-provided-vnet-subnet"]
+= Configuring a private storage endpoint on Azure with user-provided VNet and subnet names
+
+Use the following procedure to configure a storage account that has public network access disabled and is exposed behind a private endpoint on Azure.
+
+.Prerequisites
+
+* You have configured an image registry to run on Azure
+* You must know the VNet and subnet names used for your Azure environment.
+
+.Procedure 
+
+. Edit the Image Registry Operator `config` object and configure the private endpoint using your VNet and subnet names:
++
+[source,terminal]
+----
+$ oc edit configs.imageregistry/cluster
+----
++
+[source,terminal]
+----
+# ...
+spec:
+  [...]
+   storage:
+      azure:
+        [...]
+        networkAccess:
+          type: Internal
+          internal:
+            subnetName: <subnet_name>
+            vnetName: <vnet_name>
+# ...
+----
+
+. Optional. Enter the following command to confirm that the Operator has completed provisioning. This might take a few minutes. 
++
+[source,terminal]
+----
+$ oc get configs.imageregistry/cluster -o=jsonpath="{.spec.storage.azure.privateEndpointName}" -w
+----
+
+////
```

Comment:
Are we missing the "When redirect is enabled, pulling images from outside of the cluster is not expected to work." note here too?

Reply:
added